heuristic/state.py

```diff
@@ -7,7 +7,8 @@
 
 
 class State:
-    def __init__(self, decision_vars, soft_vars, hard_vars, objective_function_value, f):
+    def __init__(self, decision_vars, soft_vars, hard_vars, objective_function_value, f,
+                 hard_penalty=1):
```

Comment:
Why do you set the hard_penalties to 1? Wouldn't this be different from our current hard_penalty?

Reply:
Yeah, good catch! What is the right value? 12? 10?

Reply:
The current hard penalty constant is 12

Reply:
Fixed